src/elements/picker/extras/picker-results/PickerResults.ts

```diff
@@ -260,16 +279,30 @@ export class BasePickerResults {
             return isPreselected;
         }) !== -1;
     }
+
+    onScrollDown(event) {
+        if (event.target) {
+                let offset = event.target.offsetHeight + event.target.scrollTop,
+                    bottom = event.target.scrollHeight;
+                if (offset >= bottom) {
+                    event.stopPropagation();
+                    if (!this.lastPage) {
+                        this.processSearch();
+                    }
+                }
+            }
+    }
 }
 
 @Component({
     selector: 'picker-results',
     host: {
         'class': 'active',
-        '[hidden]': 'matches.length === 0'
+        '[hidden]': 'matches.length === 0',
+        '(scroll)': 'onScrollDown($event)'
```

Comment:
Can you move this to `@HostListener`? These will be deprecated eventually.

https://github.com/angular/angular/blob/4.0.0-beta.2/modules/%40angular/core/src/metadata/directives.ts#L981